ddpui/management/commands/createorganduser.py

```diff
@@ -40,7 +41,15 @@ def handle(self, *args, **options):
 
         # fetch / create the User
         if not User.objects.filter(email=options["email"]).exists():
-            User.objects.create_user(email=options["email"], username=options["email"])
+            password = getpass.getpass("Enter the password for the OrgUser: ")
```

Comment:
Is it possible to just pass it as an argument since we are automating the process? We can have it as a .env variable.

Reply:
nice idea, have added both